app/views/users/show.html.erb

```diff
@@ -46,6 +46,15 @@
             } %>
       </div>
     <% end %>
+    <% if !(current_user == @user) %>
```

Comment:
Why not
```suggestion
    <% if current_user.admin? %>
```

Reply:
it's not about checking the users role but more if the user that is shown is the currently logged in user